.travis.yml

```diff
@@ -6,6 +6,10 @@ rvm:
   - 2.3
   - 2.4
 
+services:
+  - mysql
+  - postgresql
```

Comment:
Before I added these services, I was getting the [CI failure](https://travis-ci.org/basecamp/marginalia/jobs/573964712)

```
mysql -u root -e "drop database if exists marginalia_test;"
ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/var/run/mysqld/mysqld.sock' (2)
```

which seems unrelated to this PR, so CI is probably broken on master